wicker/core/datasets.py

```diff
@@ -152,54 +306,50 @@ def get_folder_size(bucket, prefix):
         return get_folder_size(bucket, key)
 
     def _get_dataset_size(self):
-        """Gets total size of the dataset in bits
+        """Gets total size of the dataset in bits.
```

Comment:
@pickles-bread-and-butter it looks like the function returns the size in `bytes` (not `bits`), could you verify and make this correction (to the original source here). Kind of a funny error IMO.

Reply:
Oh yeah 🤣 , it says it in the boto docs and I flubbed the doc string here.